static/js/popovers.js

```diff
@@ -1280,6 +1286,49 @@ export function register_click_handlers() {
         e.preventDefault();
     });
 
+    $("body").on("click", ".deactivate_user_from_popup_info", (e) => {
+        hide_all();
+
+        function confirm_deactivation() {
+            $("#confirm_user_deactivation_modal").modal("hide");
+            const user_id = elem_to_user_id($(e.target).parents("ul"));
+            const url = "/json/users/" + encodeURIComponent(user_id);
+            const request_method = channel.del;
+            request_method({
+                url,
+            });
+        }
+
+        const user_id = elem_to_user_id($(e.target).parents("ul"));
+        const user = people.get_by_user_id(user_id);
+
+        $("#confirm_user_deactivation_modal_holder").html(render_confirm_user_deactivation_modal());
+        $("#confirm_user_deactivation_modal").find(".user_name").text(user.full_name);
+        $("#confirm_user_deactivation_modal").find(".email").text(user.email);
```

Comment:
You should pass these into `render_confirm_user_deactivation_modal` as context, rather than editing the content after rendering it.